lib/bot/events/temp_voice_event_container.rb

```diff
@@ -31,9 +572,15 @@ def self.handle_channel_join(event)
     discord_jump_channel =
       server.voice_channels.detect { |c| c.id == channel.id }
 
+    # Get adjective
+    adjective = adjective_list[rand(adjective_list.length)]
+
+    # Get noun
+    noun = noun_list[rand(noun_list.length)]
+
     # Make the new channel
     new_channel = server.create_channel(
-      "~#{event.user.name}",
+      adjective.capitalize + " " + noun.capitalize,
```

Comment:
Usually, in `ruby`, you'd see something like this:
```suggestion
      "#{adjective.capitalize} #{noun.capitalize}"
```

Reply:
Also, maybe more the use of `capitalize` to the variable definition?  🤔  Just to keep the string here cleaner.